src/app/features/login/login.component.scss

```diff
@@ -1,4 +1,7 @@
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css");
```

Comment:
Why do we need it if there is font awesome already in the project?

Reply:
In this case i could not make it work with font awesome and because of this i implemented bootstrap icon